packages/react/src/models/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './temporal';
```

Comment:
The Base UI team created a `/types` folder intead.
I think we can move all the logic that we want to have public in this folder.

Given that the adapter are supposed to remain internal, we will have to see what the Scheduler needs (could be exported from the utils package)

For me we  need at least those three to use internally:

- `TemporalAdapter`
- `TemporalTimezone`
- `TemporalSupportedObject`

Reply:
Moved them to `./types`: https://github.com/mui/base-ui/pull/1973/commits/bb76fdc20829f1e82a54a14af21a7d85bfab6656